### PR TITLE
Added loading indicator icons, started including the current icons with fabric

### PR DESCRIPTION
**Pull request by AlexisDeschamps**
## Why
Icons were not previously included with fabric, this meant that the importing project had to include the images itself, e.g. https://github.com/wealthsimple/terminal/tree/master/src/images.

When working on w4w-ui, I wanted to use some of these icons, which meant I would have to rely on terminal's images. This PR changes this by including the four images with fabric

## What
- Added three mixins that set SVG backgrounds. The mixins take a color parameter.
- Added three CSS classes that can be used to display the icons. They make use of the mixins with $color-gray-60.
- Added docs page describing the icons
- Updated `form-control` to use the new mixin
- Added the update source SVGs to the `images` folder

## Screenshot
image

### CODE REVIEW

File: scss/_svgs.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+$ws-icon-arrow-down: "data:image/svg+xml,%3C%3Fxml version='1.0' encoding='utf-8'%3F%3E%3C!-- Generator: Adobe Illustrator 19.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0%29 --%3E%3Csvg version='1.1' id='Layer_1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 25 16' style='enable-background:new 0 0 25 16;' xml:space='preserve'%3E%3Cstyle type='text/css'%3E .st0%7Bfill:%23A6A6A6;%7D%0A%3C/style%3E%3Cpolygon class='st0' points='12.5,15.1 0,2.6 1.8,0.9 12.5,11.6 23.2,0.9 25,2.6 '/%3E%3C/svg%3E%0A";
```
Comment (cchawn) [inline scss/_svgs.scss]: The inline SVG's should probably be cleaned up. You can remove the comments, as well as the unnecessary descriptions. You could also run the SVG's through some optimization [tools](https://jakearchibald.github.io/svgomg/) to reduce the file size 🤷‍♀️

File: scss/_variables.scss [hunk 2]
```diff
@@ -10,6 +10,7 @@ $bp-site-max-width: 1440px;
 $site-max-width: 1400px;
 
 $base-icon-size: 22px;
+$base-icon-size-large: 66px;
```
Comment (cchawn) [inline scss/_variables.scss]: You could change this to `$large-icon-size: $base-icon-size * 3;` 🤷‍♀️

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,6 @@
 .bundle/config
 bin/
+docs/.DS_STORE
```
Comment (cchawn) [inline .gitignore]: This can probably just be `.DS_STORE` (without the path to docs) 🤷‍♀️

**Comment by AlexisDeschamps** [comment]
@cchawn Thanks a lot for the review! I tried to minimize the SVGs. I also changed the icons to be mixins that take in a color parameter.

**Pull request merged**